primitives/src/parachain.rs

```diff
@@ -240,6 +242,36 @@ pub struct ConsolidatedIngress(pub Vec<(Id, Vec<Message>)>);
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
 pub struct BlockData(#[cfg_attr(feature = "std", serde(with="bytes"))] pub Vec<u8>);
 
+/// A chunk of erasure-encoded block data
+#[derive(PartialEq, Eq, Clone, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
+pub struct ErasureChunk{
+	/// The parent of the candidate this chunk belongs to.
+	pub relay_parent: Hash,
```

Comment:
suggest changing the argument types here to type aliases instead, as mentioned here https://github.com/paritytech/polkadot/pull/345/files#r308998575